torch/csrc/jit/serialization/pickler.cpp

```diff
@@ -801,4 +801,87 @@ bool checkHasValidSetGetState(const std::shared_ptr<c10::ClassType>& cls) {
   return true;
 }
 
+// Declare BackendMeta serialization and deserialization function pointer types.
+using BackendMetaPtr =
+    void (*)(const at::Tensor&, std::unordered_map<std::string, int>&);
+
+// map to save function pointer for BackendMeta serialization.
+// key is the DeviceType, value is std::pair obj. 
+// value.first represent get function and value.seconde represent set function
+static std::unordered_map<int, std::pair<void*, void*>> serialization_map;
```

Comment:
>OK, having understood the approach, I am more positive about this. Please stop using a string hash map with get/set fields though.

This variable `serialization_map` map the backend and the registered two serialization functions